src/components/layout/TwoColumnsLayout.tsx

```diff
@@ -29,14 +31,15 @@ interface Props {
   maxContentWidth?: keyof typeof sizes.container;
 }
 
-const TwoColumnsLayout: FC<Props> = ({
-  header,
-  title,
-  backLink,
-  left: { content: leftContent, columns: leftColumns = 6 },
-  right: { content: rightContent, columns: rightColumns = 6 },
-  maxContentWidth = 'lg',
-}) => {
+const TwoColumnsLayout: FC<TwoColumnsLayoutProps> = (props) => {
+  const {
+    header,
+    title,
+    left: { content: leftContent, columns: leftColumns = 6 },
+    right: { content: rightContent, columns: rightColumns = 6 },
+    maxContentWidth = 'lg',
+  } = props;
+
```

Comment:
Is there a reason for not destructuring props?

Reply:
yes. Typescript is not able to figure out the conditional type correctly when you destruct it..